createrepo_c.spec

```diff
@@ -251,11 +251,13 @@ ln -sr %{buildroot}%{_bindir}/modifyrepo_c %{buildroot}%{_bindir}/modifyrepo
 %if %{with python2}
 %files -n python2-%{name}
 %{python2_sitearch}/%{name}/
+%{python2_sitearch}/%{name}-%{version}-py2.7.egg-info/
```

Comment:
You might want to use `%{python2_version}` and `%{python3_version}` instead of `2.7` and `3.7`.  There isn't going to be a 2.8, but python3 is going to continue to increment.

Reply:
yes, thank you.